### ISSUE TITLE
Getting error - printf: bad formatting char 'v'

### ISSUE THREAD

**Issue by sean-zou** [fugue/fregot#147]
Getting error - `printf: bad formatting char 'v'`
```fregot (eval error):
  "policy/main.rego" (line 14, column 12):
  builtin error:

    14|     msg := sprintf("Failed for the following resources: %v", [resources])
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  printf: bad formatting char 'v'```

**Comment by jaspervdj-luminal** [comment]
Thanks again for reporting all of these @sean-zou, it's really valuable!  This should be fixed shortly.

**Comment by jaspervdj-luminal** [comment]
This should be fixed in `v0.7.1`.
